driver-bundle/pom.xml

```diff
@@ -37,9 +37,9 @@
   </build>
   <dependencies>
     <dependency>
-      <groupId>com.microsoft.playwright</groupId>
+      <groupId>io.github.palexdev.playwright</groupId>
```

Comment:
Revert this and similar changes in other pom.xml files

Reply:
Oh I'm very sorry for this, I didn't know the commit would be automatically added to the PR
Since I needed the library in one of my modular projects yesterday I decided to publish artifacts on my maven repo
I'll update the PR in the morning